### ISSUE TITLE
Applying graph2vec with subgraphs

### ISSUE THREAD

**Issue by SihamAmarouche** [MLDroid/graph2vec_tf#9]
Can I use graph2vec to represent subgraphs(that contain two nodes or three) as vectors then I will cluster these subgraphs to N number clusters using K-means or Dbscan algorithms Is it a logical process ?

Thanks very much .

**Comment by alexiatoumpa** [comment]
Yes that makes sense, as your clusters will be in your embedding space.
